src/app/modals/return-to-office/return-to-office.page.scss

```diff
@@ -0,0 +1,129 @@
+.modal-page {
+    --background: #f2f2f2;
+}
+.close-icon {
+    height: 50px;
+    width: 35px;
+    color: var(--color-blue);
+}
+
+.close-button {
+    display: flex;
+    justify-content: flex-end;
+    margin-right: 10px;
+}
+
+.title {
+    padding-bottom: 20px;
+    margin: 0;
+    font-size: 30px;
+    font-weight: 500;
+    letter-spacing: 1px;
+    color: var(--color-blue);
+    padding-left: 30px;
+}
+
+.sub-title {
+    margin: 0;
+    font-size: 20px;
+    font-weight: 500;
+    letter-spacing: 1px;
+    color: var(--color-blue);
+    padding-left: 30px;
+}
+
+.card-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding-bottom: 25px;
+}
+
+.menu {
+    display: inline-grid;
+    grid-template-columns: auto auto;
+}
+
+.menu-card {
+    position: relative;
+    margin: 10px;
+    max-width: 190px;
+    max-height: 130px;
+    align-self: center;
+    justify-self: center;
+    background-color: #fff;
+    width: 173px;
+    height: 126px;
+    padding: 10px 0 10px 24px;
+    font-family: "Lato", sans-serif;
+    box-shadow: 0 5px 7px 0 rgba(0, 0, 0, 0.25);
+}
+
+@media (max-width: 375px) {
```

Comment:
Note : Instead of applying media query , you can go ahead with percentage width and height.